### ISSUE TITLE
Malicious Program (Hybrid Analysis) ?

### ISSUE THREAD

**Issue by antuketot76** [winauth/winauth#636]
Is this false positive detected as malicious program by Hybrid Analysis ?

GitHub File Link - [HERE](https://github.com/winauth/winauth/releases/download/3.5.1/WinAuth-3.5.1.zip)

Hybrid Analysis Link - [HERE](https://www.hybrid-analysis.com/sample/bec21c9b8c94fdf9977eed526e730d11a89d25472f90cd87ee47dcfeddbe237d)

ViruTotal Link - [HERE](https://www.virustotal.com/#/file/eb08274c182ab8b68b375d06f35109af7c6c034f9d06c1ac81e26552d9d81d45/detection)

**Comment by Wqrld** [comment]
Yes, it's a false positive. All sites on there state that it is clean but it's still saying malicious for some reason. I have been using this for years.
